dbms/src/Server/FlashGrpcServerHolder.cpp

```diff
@@ -213,6 +213,10 @@ FlashGrpcServerHolder::~FlashGrpcServerHolder()
         int wait_cnt = 0;
         while (GET_METRIC(tiflash_object_count, type_count_of_mpptunnel).Value() >= 1 && (wait_cnt++ < max_wait_cnt))
             std::this_thread::sleep_for(std::chrono::seconds(1));
+        if (GET_METRIC(tiflash_object_count, type_count_of_mpptunnel).Value() >= 1)
+            LOG_WARNING(log, "Wait {} seconds for mpp tunnel, still some mpp tunnel is alive, potential resource leak", wait_cnt);
+        else
+            LOG_WARNING(log, "Wait {} seconds for mpp tunnel, all finished", wait_cnt);
```

Comment:
```suggestion
            LOG_INFO(log, "Wait {} seconds for mpp tunnel, all finished", wait_cnt);
```
warnings can be misleading?

Reply:
Change the log level to info